tests/test-inferences/test_metropolishastings.py

```diff
@@ -31,6 +32,15 @@ def test_normalnormal_run(self):
       self.assertAllClose(qmu.stddev().eval(), np.sqrt(1 / 51),
                           rtol=1e-2, atol=1e-2)
 
+      sess = ed.get_session()
```

Comment:
You can remove this line since the `with` context already provides the session as `sess`.